integration-tests/features/list-machines.feature

```diff
@@ -5,4 +5,5 @@ Scenario: Machine listing responsiveness
          | name       | definition          | ensure_fresh |
          | vm1        | centos6-guest-httpd | no           |
          | vm2        | centos7-target      | no           |
-   Then a full machine listing should take less than 120 seconds
+   Then a shallow machine listing should take less than 15 seconds
+    and a full machine listing should take less than 30 seconds
```

Comment:
So as it turns out, we had to revert it back to the 120 seconds - Which I did - Because on the CI neither the guest agent approach nor the virt inspector approach is available.